docs/source/first_example.md

```diff
@@ -82,7 +82,7 @@ from ase.calculators.emt import EMT
 from wfl.calculators import generic
 from wfl.autoparallelize import AutoparaInfo
 from wfl.configset import ConfigSet, OutputSpec
-from wfl.autoparallelize.remote info import RemoteInfo()
+from wfl.autoparallelize.remote.info import RemoteInfo()
```

Comment:
imports just need names, not function calls (i.e. no `()`)

Reply:
And even `remote.info` isn't correct.. Fixed this and other on the same page.